### ISSUE TITLE
test Canto has genotypes in physical interaction dialog

### ISSUE THREAD

**Issue by mah11** [pombase/canto#2176]
e.g. screenshot from https://curation.pombase.org/test/curs/fa1b44d12d7d1106 at time of reporting:

![Screen Shot 2020-01-08 at 11 21 42 AM](https://user-images.githubusercontent.com/5196259/71975192-32691d00-320b-11ea-8f4c-6bbc4503c1da.png)

From the #2115 summary it looks like that ticket (and linked comments) reported the same problem, so maybe the fix just needs to be rolled out to the 'test' Canto instance.

**Comment by kimrutherford** [comment]
Yep, the code is fixed now.  I've removed the existing physical interaction annotations from the test database as they were all screwed up.  You can now make gene-gene interactions again.
